Task_Position_the_Cubes/index.css

```diff
@@ -1,46 +1,58 @@
 /*  Start coding below for POSITION THE CUBES TASK*/
-.parent {
+.flexbox-container {
   width: 300px;
   height: 300px;
-  /* padding: 80px 50px;
-  padding-bottom: 80px;
-  padding: left 30px;;
-  border: 5px solid gray;
-  border-radius: 0px;*/
```

Comment:
The 'position: relative;' property is unnecessary since the default value for position is already static.